src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -128,4 +129,10 @@ public User getUser(String email) {
 	}
 	return users;
   }
+  
+  public int getTotalMessageCount(){
```

Comment:
nit: add space between () a d {

Reply:
Thank you for the review! I have updated on stats_update branch!